lib/logcache/client.rb

```diff
@@ -44,8 +44,14 @@ def container_metrics(source_guid:, start_time:, end_time:, envelope_limit: DEFA
 
     private
 
-    def with_request_error_handling(_source_guid)
+    def with_request_error_handling(source_guid)
       tries ||= 3
+      start_time = Time.now
+      time_taken = (Time.now - start_time) * 1000 # convert to milliseconds to get more precise information
+      logger.info("Response time: #{time_taken.round(2)} ms",
```

Comment:
Other client's logs look rather like `lrp.instances.request`, so I'd prefer the actual log to look like `logcache.response`. `event` could then be removed from the hash.

Reply:
No I meant replacing the log message `"Response time: #{time_taken.round(2)} ms"` with `"logcache.response"`.

Reply:
I've already updated the code based on your review